cardano-testnet/test/Main.hs

```diff
@@ -34,6 +35,7 @@ tests = do
       , H.testProperty "Spec.Plutus.SubmitApi.TxInLockingPlutus" Spec.Plutus.SubmitApi.TxInLockingPlutus.hprop_plutus
       , ignoreOnWindows "Spec.Plutus.Direct.ScriptContextEquality"  Spec.Plutus.Direct.ScriptContextEquality.hprop_plutus_script_context_equality
       , ignoreOnWindows "Spec.Plutus.Direct.ScriptContextEqualityMint" Spec.Plutus.Direct.ScriptContextEqualityMint.hprop_plutus_script_context_mint_equality
+      , ignoreOnWindows "Spec.Shutdown" Spec.Shutdown.hprop_shutdown
```

Comment:
Just put a comment saying that currently there is a blocking call on Windows that prevents graceful shutdown (or something to that effect). And that we currently aren't testing the shutdown IPC flag.

Reply:
Done!